scripts/src/main/resources/scripts/command/ide

```diff
@@ -306,27 +306,28 @@ function doUninstall() {
 
 function doUpdateProjects() {
   for projectconfig in "${DEVON_IDE_HOME}"/settings/projects/*.properties; do
-    echo "Importing from ${projectconfig}..."
-    project_path=$(grep "^path" "$projectconfig" | cut -d'=' -f2)
-    project_workingsets=$(grep "^workingsets" "$projectconfig" | cut -d'=' -f2)
-    project_workspace=$(grep "^workspace" "$projectconfig" | cut -d'=' -f2)
-    project_git_url=$(grep "^git.url" "$projectconfig" | cut -d'=' -f2)
-    project_git_branch=$(grep "^git.branch" "$projectconfig" | cut -d'=' -f2)
-    project_eclipse=$(grep "^eclipse" "$projectconfig" | cut -d'=' -f2)
+    doEcho "Importing from ${projectconfig}..."
+    local project_path=$(grep "^path" "$projectconfig" | cut -d'=' -f2)
+    local project_workingsets=$(grep "^workingsets" "$projectconfig" | cut -d'=' -f2)
+    local project_workspace=$(grep "^workspace" "$projectconfig" | cut -d'=' -f2)
+    local project_git_url=$(grep "^git.url" "$projectconfig" | cut -d'=' -f2)
+    local project_git_branch=$(grep "^git.branch" "$projectconfig" | cut -d'=' -f2)
+    local project_eclipse=$(grep "^eclipse" "$projectconfig" | cut -d'=' -f2)
     
     [ -z "${project_path}" ] && doFail "Config is missing required parameter 'project.path'"
-    [ -z "${project_workspace}" ] && local project_workspace=main
+    [ -z "${project_workspace}" ] && project_workspace=main
     [ -z "${project_git_url}" ] && doFail "Config is missing required parameter 'project.git.url'"
-    [ -z "${project_eclipse}" ] && local project_eclipse=import
+    [ -z "${project_eclipse}" ] && project_eclipse=import
     
-    echo "Project (Path: ${project_path}, Url: ${project_git_url}, Branch: ${project_git_branch}, Eclipse: ${project_eclipse}, Workspace: ${project_workspace}, Workingsets: ${project_workingsets})"
-    if [ "${project_workspace}" == "${WORKSPACE}" ]
```

Comment:
why did you remove the if-statement?  now all projects will be imported into all workspaces...

Reply:
No each project is configured into the workspace that is configured.
Otherwise only the projects for the main workspace where perocessed during setup and all others had been ignored.

Reply:
OK. We now have clarified this in a call. Thanks as this revealed gaps on both of our sides so we can now fix everything properly.